src/app/tests/suites/TestArmFailSafe.yaml

```diff
@@ -30,8 +30,14 @@ tests:
       command: "Reboot"
       arguments:
           values:
+              - name: "registerKey"
```

Comment:
Is it possible to have this default to "default" so the vast majority of tests, which are using a single app, don't have to deal with it?

And ideally we would not need to pass port/kvs for these things; those args should be coming from the harness, no?

Reply:
As discussed offline, addressed this by making `Reboot` take only one optional arg (registerKey which defaults to "default"). The meaning of `Reboot` shall be to reboot the app indicated by the registerKey and  use whatever command line option the application was last started with.